pom.xml

```diff
@@ -81,7 +81,7 @@
             <!--Overriding spring-web version to remove vulnerability-->
             <groupId>org.springframework</groupId>
             <artifactId>spring-web</artifactId>
```

Comment:
This spring web override can be removed.

Reply:
As per slack, resolving this on the basis that a fairly new CVE is present in 6.1.11 which is the version pulled in unless overridden